TFLite_detection_stream.py

```diff
@@ -92,6 +94,47 @@ def index():
         return make_response('Could not verify!', 401, {'WWW-Authenticate' : 'Basic realm="Login Required"'})
     return render_template("index.html")
 
+# MOVE TO APPLICATION MODULE
+# Define the custom Jinja filter
+@app.template_filter()
+def zip_lists(a, b, c):
+    return zip(a, b, c)
+
+# MOVE TO APPLICATION MODULE
+# Register the custom filter with Jinja
+app.jinja_env.filters['zip'] = zip_lists
+
+# MOVE TO APPLICATION MODULE
+@app.route("/settings")
+def settings():
+    status = []
+    recipients = vault.get_keys("recipients")
+    numbers = []
+    for key in recipients:
+        status.append(int(vault.get_value("recipients", key, "active")))
+        numbers.append("XXX-XXX-" + vault.get_value("recipients", key, "phone_number")[-4:])
+    return render_template("settings.html", users=recipients, statuses=status, phone_nums=numbers)
+
+# MOVE TO APPLICATION MODULE
+@app.route('/users/<string:user>/sms-notifications', methods=["PUT"])
+def update_sms_status(user):
+    data = request.get_json()
+    sms_status = int(data['sms_notifications'])
+
+    # Update the user's record in the database
+    vault.put_value("recipients", user, "active", str(sms_status))
```

Comment:
In the future we will make specific functions for putting values, so in this case; `put_status` or `put_sms_status` :D 
Ref: https://trello.com/c/22FPrkYW